django_rq/views.py

```diff
@@ -84,6 +92,7 @@ def jobs(request, queue_index):
         'page': page,
         'page_range': page_range,
         'job_status': 'Queued',
+        'jquery_path': JQUERY_PATH,
```

Comment:
Instead of sprinkling jquery_path variable everywhere, can we just write a simple template tag that outputs the right path?